### PR TITLE
Recipient address validation and more improvements.

### PR DESCRIPTION
**Pull request by LogicSatinn**
- Adding a dedicated class for validating the phone addresses that prevents sending bad requests.
- Automatically check and change the phone addresses where necessary.
- New method unpackRecipients that takes advantage of the spread operator in PHP.

### CODE REVIEW

File: src/Classes/Validator.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+use Illuminate\Support\Str;
+
+final class Validator
+{
+    /**
+     * List of phone address prefixes.
+     *
+     * @var array<string>
+     */
+    protected static array $phoneAddressPrefix = [
```
Comment (JemCdo) [inline src/Classes/Validator.php:18]: @LogicSatinn can we move this to config file ili iwe dynamic for users?? ... let's say for outside tz

File: src/Classes/Validator.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        return self::$phoneAddressPrefix;
+    }
+
+    /**
+     * @return Validator
+     */
+    protected function fixIfPhoneAddressStartsWithZeroOrPlus(): Validator
+    {
+        $this->phoneAddresses = Arr::map($this->phoneAddresses, static function ($phoneAddress) {
+            if (Str::startsWith($phoneAddress, ['07', '06'])) {
```
Comment (JemCdo) [inline src/Classes/Validator.php:72]: @LogicSatinn also here though will me mauno

**Review by JemCdo — CHANGES_REQUESTED** [review]

File: src/Classes/Validator.php (hunk 1 above)
Comment (LogicSatinn) [reply src/Classes/Validator.php:18]: @JemCdo Do Beem send messages to outside TZ ??

I don't think they do.

File: src/Classes/Validator.php (hunk 2 above)
Comment (LogicSatinn) [reply src/Classes/Validator.php:72]: Pardon??

File: src/Classes/Validator.php (hunk 1 above)
Comment (JemCdo) [reply src/Classes/Validator.php:18]: They do nini?

Comment (JemCdo) [reply src/Classes/Validator.php:18]: They do ... angalia coverage yao on their website

Comment (LogicSatinn) [reply src/Classes/Validator.php:18]: They actually do send messages internationally.... but I think they use a different set of APIs.

Comment (JemCdo) [reply src/Classes/Validator.php:18]: not sure, I will merge this for now

**Pull request merged**